spring-cloud-previews/google-cloud-artifact-registry-spring-starter/src/main/java/com/google/devtools/artifactregistry/v1/spring/ArtifactRegistrySpringAutoConfiguration.java

```diff
@@ -170,6 +170,46 @@ public ArtifactRegistrySettings artifactRegistrySettings(
               clientSettingsBuilder.getDockerImageSettings().getRetrySettings(), serviceRetry);
       clientSettingsBuilder.getDockerImageSettings().setRetrySettings(getDockerImageRetrySettings);
 
+      RetrySettings listMavenArtifactsRetrySettings =
```

Comment:
This correspond to changes from https://github.com/googleapis/google-cloud-java/pull/9080